README.md

```diff
@@ -42,6 +42,22 @@ files. Until IntelliJ is restarted, the `Reformat code` action will also
 There is an [open bug](https://devnet.jetbrains.com/thread/464297) against
 IntelliJ to add support for configuring external formatters.
 
+### Gradle
+
+[Spotless](https://github.com/diffplug/spotless) can apply [google-java-format](https://github.com/diffplug/spotless#applying-to-java-source-google-java-format) automatically to all java source in a project, as well as apply other kinds of formatter to other kinds of files.
+
+```gradle
+plugins {
+   id 'com.diffplug.gradle.spotless' version '2.2.0'
+}
+
+spotless {
+    java {
+        googleJavaFormat()
```

Comment:
Providing published and SNAPSHOT versions works as well:
- `googleJavaFormat('1.1')`
- `googleJavaFormat('1.2-SNAPSHOT')`